task: primeira versão dos serviços da api

br-rafaelbarros opened the pull request:
Implementado seguindo alguns exemplos do openapi 3.0.0

br-rafaelbarros committed:
primeira versao dos serviços da api

mateuslima90 commented:
Rafael, 
A tua especificação ficou muito boa, bem explicativa e fácil entendimento. Anotei várias melhorias para minha especificação com alguns itens que tu utilizou. O único ponto, como tu mesmo falou sobre api de get ter paginação. 
Parabéns.

br-rafaelbarros committed:
fix: ajustes security, pagintation

ajustado nomenclatura em requisições, ajustado chavez de autenticação e paginação em get
